### PR TITLE
Fix :attr:`.Axes.coordinate_labels` referring to the entire axis, not just its labels.

### PR DESCRIPTION
**Pull request by hydrobeam**
## Overview: What does this pull request change?

`Axes.coordinate_labels` used to refer to the entirety of the axis since `NumberLine.add_numbers/labels` return `self`. Setting `labels` within `add_coordinates` to explicitly refer to the numbers/labels produced by these methods fixes this issue.

## Motivation and Explanation: Why and how do your changes improve the library?

```py
class AxCoords(Scene):
    def construct(self):
        a = Axes().add_coordinates()
        a.coordinate_labels.set_color(PURPLE_B)
        self.add(a)
```
After:
AxCoords_ManimCE_v0 9 0

Before:
image

## Links to added or changed documentation pages
 
https://manimce--1967.org.readthedocs.build/en/1967/reference/manim.mobject.coordinate_systems.CoordinateSystem.html?highlight=add_coordinates#manim.mobject.coordinate_systems.CoordinateSystem.add_coordinates

## Reviewer Checklist
- [x] The PR title is descriptive enough for the changelog, and the PR is labeled correctly
- [ ] If applicable: newly added non-private functions and classes have a docstring including a short summary and a PARAMETERS section
- [ ] If applicable: newly added functions and classes are tested

### CODE REVIEW

**Review by ryanmccauley211 — COMMENTED** [review]
Tested locally and looks good! I was checking the docs produced and I am not sure if they are formatted correctly, it comes out like `:param : :type : ivar coordinate_labels: A VGroup of the added labels.` but not sure if they were supposed to be formatted like the other doc strings?

**Comment by hydrobeam** [comment]
Ah, I misinterpreted how to add docs for the attribute in the initial PR, it should be fixed now.

**Review by ryanmccauley211 — APPROVED** [review]
Looks good!

File: manim/mobject/coordinate_systems.py [hunk 1]
```diff
@@ -275,7 +275,7 @@ def add_coordinates(
         ],
         **kwargs,
     ):
-        """Adds labels to the axes.
+        """Adds labels to the axes. Use ``Axes.coordinate_labels`` to access the coordinates after creation.
```
Comment (behackl) [inline manim/mobject/coordinate_systems.py:278]:
```suggestion
        """Adds labels to the axes. Use ``Axes.coordinate_labels`` to
        access the coordinates after creation.
```

**Review by behackl — APPROVED** [review]
Left a suggestion for shortening a line in the docstring that seemed a bit too long, LGTM otherwise.

**Pull request merged**